tests/rptest/services/redpanda.py

```diff
@@ -129,30 +130,7 @@ def stop_node(self, node):
 
     def clean_node(self, node):
         node.account.kill_process("redpanda", clean_shutdown=False)
-        node.account.remove(RedpandaService.PERSISTENT_ROOT)
```

Comment:
it doesn't look like this cleans up the config file now that it is in a new location

Reply:
true, I'll address that.

Reply:
I pushed a change to address this